neo/SmartContract/InteropService.cs

```diff
@@ -343,7 +315,8 @@ private static bool Blockchain_GetBlock(ApplicationEngine engine)
             else
             {
                 Block block = engine.Snapshot.GetBlock(hash);
-                engine.CurrentContext.EvaluationStack.Push(StackItem.FromInterface(block));
+                if (block == null) return false;
```

Comment:
if we return `byte[0]` is the same as `null` right?

Reply:
Yes.